joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50


srhand_joint_controller: 
  type: position_controllers/JointTrajectoryController
  joints:
    - lh_FFJ4
    - lh_FFJ3
    - lh_FFJ2
    - lh_FFJ1
    - lh_MFJ4
    - lh_MFJ3
    - lh_MFJ2
    - lh_MFJ1
    - lh_RFJ4
    - lh_RFJ3
    - lh_RFJ2
    - lh_RFJ1
    - lh_LFJ5
    - lh_LFJ4
    - lh_LFJ3
    - lh_LFJ2
    - lh_LFJ1
    - lh_THJ5
    - lh_THJ4
    - lh_THJ3
    - lh_THJ2
    - lh_THJ1
    - rh_FFJ4
    - rh_FFJ3
    - rh_FFJ2
    - rh_FFJ1
    - rh_MFJ4
    - rh_MFJ2
    - rh_MFJ3
    - rh_MFJ1
    - rh_RFJ4
    - rh_RFJ3
    - rh_RFJ2
    - rh_RFJ1
    - rh_LFJ5
    - rh_LFJ4
    - rh_LFJ3
    - rh_LFJ2
    - rh_LFJ1
    - rh_THJ5
    - rh_THJ4
    - rh_THJ3
    - rh_THJ2
    - rh_THJ1

  /gazebo_ros_control/pid_gains: 
    lh_FFJ4: {p: 10.0, i: 0.01, d: 0.010}
    lh_FFJ3: {p: 10.0, i: 0.01, d: 0.010}
    lh_FFJ2: {p: 10.0, i: 0.01, d: 0.010}
    lh_FFJ1: {p: 10.0, i: 0.01, d: 0.010}
    lh_MFJ4: {p: 10.0, i: 0.01, d: 0.010}
    lh_MFJ3: {p: 10.0, i: 0.01, d: 0.010}
    lh_MFJ2: {p: 10.0, i: 0.01, d: 0.010}
    lh_MFJ1: {p: 10.0, i: 0.01, d: 0.010}
    lh_RFJ4: {p: 10.0, i: 0.01, d: 0.010}
    lh_RFJ3: {p: 10.0, i: 0.01, d: 0.010}
    lh_RFJ2: {p: 10.0, i: 0.01, d: 0.010}
    lh_RFJ1: {p: 10.0, i: 0.01, d: 0.010}
    lh_LFJ5: {p: 10.0, i: 0.01, d: 0.010}
    lh_LFJ4: {p: 10.0, i: 0.01, d: 0.010}
    lh_LFJ3: {p: 10.0, i: 0.01, d: 0.010}
    lh_LFJ2: {p: 10.0, i: 0.01, d: 0.010}
    lh_LFJ1: {p: 10.0, i: 0.01, d: 0.010}
    lh_THJ5: {p: 10.0, i: 0.01, d: 0.010}
    lh_THJ4: {p: 10.0, i: 0.01, d: 0.010}
    lh_THJ3: {p: 10.0, i: 0.01, d: 0.010}
    lh_THJ2: {p: 10.0, i: 0.01, d: 0.010}
    lh_THJ1: {p: 10.0, i: 0.01, d: 0.010}
    rh_FFJ4: {p: 10.0, i: 0.01, d: 0.010}
    rh_FFJ3: {p: 10.0, i: 0.01, d: 0.010}
    rh_FFJ2: {p: 10.0, i: 0.01, d: 0.010}
    rh_FFJ1: {p: 10.0, i: 0.01, d: 0.010}
    rh_MFJ4: {p: 10.0, i: 0.01, d: 0.010}
    rh_MFJ3: {p: 10.0, i: 0.01, d: 0.010}
    rh_MFJ2: {p: 10.0, i: 0.01, d: 0.010}
    rh_MFJ1: {p: 10.0, i: 0.01, d: 0.010}
    rh_RFJ4: {p: 10.0, i: 0.01, d: 0.010}
    rh_RFJ3: {p: 10.0, i: 0.01, d: 0.010}
    rh_RFJ2: {p: 10.0, i: 0.01, d: 0.010}
    rh_RFJ1: {p: 10.0, i: 0.01, d: 0.010}
    rh_LFJ5: {p: 10.0, i: 0.01, d: 0.010}
    rh_LFJ4: {p: 10.0, i: 0.01, d: 0.010}
    rh_LFJ3: {p: 10.0, i: 0.01, d: 0.010}
    rh_LFJ2: {p: 10.0, i: 0.01, d: 0.010}
    rh_LFJ1: {p: 10.0, i: 0.01, d: 0.010}
    rh_THJ5: {p: 10.0, i: 0.01, d: 0.010}
    rh_THJ4: {p: 10.0, i: 0.01, d: 0.010}
    rh_THJ3: {p: 10.0, i: 0.01, d: 0.010}
    rh_THJ2: {p: 10.0, i: 0.01, d: 0.010}
    rh_THJ1: {p: 10.0, i: 0.01, d: 0.010}
    

